src/store/sagas/accounts.ts

```diff
@@ -1248,12 +1250,24 @@ function* blindRefreshWorker() {
         blindRefresh: true,
       },
     }
-
+    
     const deltaTxs: TransactionDescribing[] = yield call( fetchBalanceTxWorker, {
       payload
     } )
+   // console.log("deltaTxs",deltaTxs);
+    const rescanTxs : RescannedTransactionData[]= []
```

Comment:
We should not be firing rescanSucceeded for an individual account during blind sync, we should fire it instead of the return statement w/ netDeltaTxs